tests/components/rfxtrx/test_sensor.py

```diff
@@ -343,3 +344,11 @@ async def test_rssi_sensor(hass, rfxtrx):
     state = hass.states.get("sensor.ac_213c7f2_48_rssi_numeric")
     assert state
     assert state.state == "-72"
+
+
+async def test_empty_conversions(hass, rfxtrx):
```

Comment:
i would prefer this test not direct call fhe sensor file but rather created a situation/mock that cause code to be run and result in state being some value